packages/view/backend/sirius-components-view-form-edit/src/main/java/org/eclipse/sirius/components/view/form/provider/CheckboxDescriptionStyleItemProvider.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*******************************************************************************
```

Comment:
This ********** should not be added for the generated code. Is IntelliJ doing that? ********** have also been added in several other files.

Reply:
Actually we use `*******...` for all non-EMF generated files. Maybe we should be more consistant and use that format also for EMF generated files?